test/built-ins/Array/prototype/group/emptyList.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.group
+description: Array.prototype.group populates object with correct keys and values
```

Comment:
This description doesn't quite seem to fit the test; maybe something like this?
```suggestion
description: Callback is not called and object is not populated if the array is empty
```